src/unix/newlib/mod.rs

```diff
@@ -598,10 +631,18 @@ pub const NI_NAMEREQD: ::c_int = 4;
 pub const NI_NUMERICSERV: ::c_int = 0;
 pub const NI_DGRAM: ::c_int = 0;
 
-pub const EAI_FAMILY: ::c_int = -303;
-pub const EAI_MEMORY: ::c_int = -304;
-pub const EAI_NONAME: ::c_int = -305;
-pub const EAI_SOCKTYPE: ::c_int = -307;
+pub const EAI_AGAIN: ::c_int = 2;
```

Comment:
By moving these to `mod.rs`, these are now defined (correctly) for ESP-IDF as well, thanks!